module_4/db/index.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+import fs from 'fs';
+import Sequelize from 'sequelize';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import dbConfig from '../config/dbConfig.cjs';
+
+class DB {
+  constructor() {
+    this.__filename = fileURLToPath(import.meta.url);
+    this.config = dbConfig;
+    this.connection = {};
+  }
+
+  async init() {
```

Comment:
looks like it can be split in multiple Single responsible private methods and/or helpers